CONTRIBUTING.md

```diff
@@ -172,6 +182,22 @@ balance_holder = AccountBalanceHolder()
 service = RaidenService()
 ```
 
+We try to follow a consistent naming convention throughout the codebase to make it easy for the reader of the code to understand what is going on.
+Thus we introduce the following rules:
+
+For addresses:
+
+- `<name>_address_hex` for hex encoded addresses
+- `<name>_address` for binary encoded addresses
+
+Lists of objects:
+
+- `<name>s`, e.g. `channels` for a list `Channel` object instances.
+
+Mappings/dicts:
+
+`<name>s_<name>s`, e.g. `tokenaddresses_taskmanagers`
```

Comment:
yeah, good to use the example here, since the convention really is `<name><plural>_<name><plural>` instead of just appending an `s`.